de/cheatsheet-machine-learning-tips-and-tricks.md

```diff
@@ -1,285 +1,285 @@
 
 <br>
 
 **37. Variance ― The variance of a model is the variability of the model prediction for given data points.**
 
-&#10230;
+&#10230; Varianz - Die Varianz des Modells ist die Schwankung der Modellprognose für den verwendeten Datensatz.
 
 <br>
 
 **38. Bias/variance tradeoff ― The simpler the model, the higher the bias, and the more complex the model, the higher the variance.**
 
-&#10230;
+&#10230; Verzerrung-Varianz-Dilemma - Umso einfacher das Modell, desto höher ist die Verzerrung. Umso komplexer das Modell, desto höher ist die Varianz. 
 
 <br>
 
 **39. [Symptoms, Regression illustration, classification illustration, deep learning illustration, possible remedies]**
 
-&#10230;
+&#10230; [Symptome, Regressionsabbildung, Klassifikationsabbildung, Deep Learning Beispiel, Mögliche Lösungen]
 
 <br>
 
 **40. [High training error, Training error close to test error, High bias, Training error slightly lower than test error, Very low training error, Training error much lower than test error, High variance]**
 
-&#10230;
+&#10230; [Hohe Trainingsfehlerrate, Trainingsfehler nahe am Testfehler, Hohe Verzerrung, Trainingsfehler geringer als Testfehler, Sehr geringer Trainingsfehler, Trainingsfehler erheblich geringer als Testfehler, Hohe Varianz]
 
 <br>
 
 **41. [Complexify model, Add more features, Train longer, Perform regularization, Get more data]**
 
-&#10230;
+&#10230; [Modellkomplexität, Ergänzung von Merkmalen, Längere Trainingszeit, Regularisierung, Erweiterung des Datensatzes]
 
 <br>
 
 **42. Error analysis ― Error analysis is analyzing the root cause of the difference in performance between the current and the perfect models.**
 
-&#10230;
+&#10230; Fehleranalyse - Bei der Fehleranalyse wird der Ursache der Differenz zwischen der Leistung des korrektem und des derzeitigen Modelles analysiert.
 
 <br>
 
 **43. Ablative analysis ― Ablative analysis is analyzing the root cause of the difference in performance between the current and the baseline models.**
 
-&#10230;
+&#10230; Ablative Analyse - Bei der ablativen Analyse wird die Ursache der Differenz zwischen der Leistung der Baseline und des derzeitigen Modelles analysiert.
 
 <br>
 
 **44. Regression metrics**
 
-&#10230;
+&#10230; Regressions Metriken
 
 <br>
 
 **45. [Classification metrics, confusion matrix, accuracy, precision, recall, F1 score, ROC]**
 
-&#10230;
+&#10230; [Klassifikationsbewertung, Konfusionsmatrix, Treffergenauigkeit, Genauigkeit, Trefferquote, F-Maß, ROC]
 
 <br>
 
 **46. [Regression metrics, R squared, Mallow's CP, AIC, BIC]**
 
-&#10230;
+&#10230; [Regressionsbewertung, R Quadrat, Mallow's CP Statistik, AIC, BIC (SBC)]
 
 <br>
 
 **47. [Model selection, cross-validation, regularization]**
 
-&#10230;
+&#10230; [Modellauswahl, Kreuzvalidierung, Regularisierung]
 
 <br>
 
 **48. [Diagnostics, Bias/variance tradeoff, error/ablative analysis]**
 
-&#10230;
+&#10230; [Diagnose, Verzerrung-Varianz-Dilemma, Fehler/Ablativ Analyse]
```

Comment:
"[**Modellevaluierung**, Verzerrung-Varianz-Dilemma, Fehler/Ablativ-Analyse]"

"Modellevaluierung" as in 35.?

Please also note hyphen between "Fehler/Ablativ" and "Analyse". Could not make it bold.